.github/workflows/ci.yml

```diff
@@ -41,8 +41,8 @@ on:
         required: false
         type: number
         default: 50
-      run_dependency_convergence:
-        description: "Whether to run the dependency convergence check"
+      run_sanity_checks:
```

Comment:
This name is too generic and a term we haven't used anywhere else in our CI setup.

`skip_qa_plugins_and_archunit_tests` would be more descriptive, but you can also immediately tell that it is wildly inaccurate because most QA plugins are still being run..

Reply:
ok. This will be solved when separating the test parameters (see other comments)